src/main/resources/view/MainWindow.fxml

```diff
@@ -38,26 +42,75 @@
           </Menu>
         </MenuBar>
 
-        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
-          <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
-          </padding>
-        </StackPane>
+        <TabPane fx:id="mainTabPane" tabClosingPolicy="UNAVAILABLE" VBox.vgrow="ALWAYS">
+          <Tab fx:id="home" text="Home">
+            <content>
+              <HBox>
+                <VBox>
+                  <Label styleClass="tuitionTitle" text="Reminders" prefWidth="${screen.visualBounds.width}" minWidth="250" textAlignment="CENTER" />
+                  <StackPane fx:id="upcomingTuitionListPanelPlaceholder" prefWidth="${screen.visualBounds.width}" minWidth="250" HBox.hgrow="ALWAYS"/>
+                  <padding>
+                    <Insets top="10" right="10" bottom="10" left="10" />
+                  </padding>
+                </VBox>
+
+                <VBox>
+                  <Label styleClass="tuitionTitle" prefWidth="${screen.visualBounds.width}" minWidth="250" text="Fees" />
+                  <StackPane fx:id="feePanelPlaceholder" prefWidth="${screen.visualBounds.width}" minWidth="250" style="-fx-wrap-text: true" />
+                  <padding>
+                    <Insets top="10" right="10" bottom="10" left="10" />
+                  </padding>
+                </VBox>
+              </HBox>
+            </content>
+          </Tab>
+
+          <Tab fx:id="tuition" text="Tuition">
+            <content>
+              <HBox fx:id="studentList" styleClass="pane-with-border" minWidth="500" prefWidth="${screen.visualBounds.width}" VBox.vgrow="ALWAYS">
+
+                <VBox>
+                  <Label styleClass="tuitionTitle" text="Students" prefWidth="${screen.visualBounds.width}" minWidth="250" textAlignment="CENTER" />
+                  <StackPane fx:id="studentListPanelPlaceholder" prefWidth="${screen.visualBounds.width}" minWidth="250" HBox.hgrow="ALWAYS"/>
+                  <padding>
+                    <Insets top="10" right="10" bottom="10" left="10" />
+                  </padding>
+                </VBox>
+
+                <VBox>
+                  <Label styleClass="tuitionTitle" prefWidth="${screen.visualBounds.width}" minWidth="250" text="Sessions" />
+                  <StackPane fx:id="tuitionListPanelPlaceholder" prefWidth="${screen.visualBounds.width}" minWidth="250" style="-fx-wrap-text: true" />
+                  <padding>
+                    <Insets top="10" right="10" bottom="10" left="10" />
+                  </padding>
+                </VBox>
+              </HBox>
+            </content>
+          </Tab>
+
+          <Tab fx:id="calendar" text="Calendar">
+            <content>
+
+            </content>
+          </Tab>
+
+          <Tab fx:id="time" disable="true" styleClass="time" >
```

Comment:
So the time is a tab itself 🤣 
This is not urgent, but perhaps we could shift the date/time to the extreme right? Would look nicer!

Reply:
HAHA ya... T_T I tried damn long to make it go to the extreme right but I just couldn't do it. Maybe I'll create an issue for anyone who wants to try 😆

Reply:
Perhaps don't make it a tab! But yes can make an issue, not urgent!